_posts/2018-09-13-Rust-1.29.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: "Announcing Rust 1.29"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.29.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.29.0 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.29.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1290-2018-09-13
+
+## What's in 1.29.0 stable
+
+The 1.29 release is fairly small; Rust 1.30 and 1.31 are going to have a lot
+in them, and so much of the 1.29 cycle was spent preparing for those
+releases. The two most significant things in this release aren't even language
+features: they're new abilities that Cargo has grown, and they're both about lints.
+
+* `cargo fix` can automatically fix your code that has warnings
+* `cargo clippy` is a bunch of lints to catch common mistakes and improve your Rust code
```

Comment:
this is probably a product call that is not worth debating on this PR, but both of these deal with lints... we might want to specify that cargo fix won't fix cargo clippy lints (i am assuming that is tru)

Reply:
Seems like this is specified on line 147-148.